README.md

```diff
@@ -64,25 +66,41 @@ included _"Test Pull Requests"_ Github Action.
 
 ## Building a local development Assistant
 
-The quickest way to get the Assistant running in Sketch is to build it locally and add it directly
-to a Sketch document.
+The next quickest way to get the Assistant running is to build it locally and add it directly to a
+Sketch document.
 
 1. Run `yarn package-tarball`.
 1. Note that a `*.tgz` tarball has been built to the repository root.
 1. Use the _Install from disk_ option in Sketch's Assistants menu to add the freshly built Assistant
    to a document.
+1. Sketch will check the disk location for updates to the tarball, so you can continue to re-build
+   the Assistant to the same location and see live updates in Sketch
 
-## Deploying the Assistant
-
-In order for others to use your Assistant the tarball produced via the `yarn package-tarball`
-command needs to be hosted publically on the web.
+> ⚠️ Sketch documents with local Assistants added to them are not portable. That is, if you share a
+> Sketch document with a local Assistant with a co-worker they will not have access to the same
+> Assistant. If you need a shareable Sketch document with Assistants then you'll need to deploy your
+> Assistant publically. Read the next section for a guide on doing this.
```

Comment:
I would inverse the argument, pointing out the benefits of publishing before explaining why the local ones won't work.

```suggestion
> ⚠️ Assistants must be published to be used across multiple devices or within a team. Sketch documents with local Assistants added to them are not portable because the Assistant is referenced using a file path.
```

Since the next paragraph is about publishing, we can ket them just read on.